js/inputbar.js

```diff
@@ -294,6 +294,14 @@ weechat.directive('inputBar', function() {
                     return true;
                 }
 
+                if ($event.altKey && (code === 38 || code === 40)) {
+                    $event.preventDefault();
+                    var direction = code - 39;
+                    console.log('direction:', direction)
```

Comment:
please remove the debug logging. I use it a lot myself during development but we should keep it clean in production code (maybe we could use $log with appropriate levels but that's another discussion)